README.md

```diff
@@ -13,21 +19,12 @@
 
  [`windowsservercore-1803, latest` (*windowsserver/Dockerfile*)](https://github.com/Microsoft/iis-docker/blob/master/windowsservercore-1803/Dockerfile)
 
-# Supported Windows Server, version 1709 amd64 tags
-
- `docker pull mcr.microsoft.com/windows/servercore/iis:windowsservercore-1709`
-
- [`windowsservercore-1709, latest` (*windowsserver/Dockerfile*)](https://github.com/Microsoft/iis-docker/blob/master/windowsservercore-1709/Dockerfile)
-
 # Supported Windows Server 2016 amd64 tags
 
  `docker pull mcr.microsoft.com/windows/servercore/iis:windowsservercore-ltsc2016`
 
  [`windowsservercore-ltsc2016, latest` (*windowsserver/Dockerfile*)](https://github.com/Microsoft/iis-docker/blob/master/windowsservercore-ltsc2016/Dockerfile)
 
- `docker pull mcr.microsoft.com/windows/servercore/iis:nanoserver-sac2016`
-
-## What is IIS?
```

Comment:
Looks like this line was erroneously deleted

Reply:
crud, yep, I'll fix it

Reply:
Fixed